CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+### 1.75 - 2020-09-08
```

Comment:
Not sure what I was supposed to do here, but I assumed this would be updated to the appropriate version/date before merging.

Reply:
@brianpmaher yeah, I think that this will be completed finally from the Cesium team 😃